### PR TITLE
Add support for standalone tools integration

### PR DESCRIPTION
**Pull request by daniviga**
Also does a general refactorization of the `opt` (Python included) and `env` (for macOS, moder Linux) build and install scripts.

The installation time of the `opt` version is cut by half. Both installers are now modular.

Tests:
- https://travis-ci.org/gem/oq-builders/jobs/270846624

### CODE REVIEW

File: installers/unix/env/install.sh [hunk 1]
```diff
@@ -79,39 +79,54 @@ check_dep $PYTHON
...
-echo "Installing the files in $FDEST. Please wait."
+source ${FDEST}/env.sh
+echo "Installing the OpenQuake Engine. Please wait."
 # Update pip first
 /usr/bin/env pip install --disable-pip-version-check -U wheelhouse/pip*.whl > /dev/null
 /usr/bin/env pip install --disable-pip-version-check wheelhouse/*.whl > /dev/null
+mkdir -p $FDEST/share
+cp -R src/{README.md,LICENSE,demos,doc} $FDEST/share
 
+## Tools installation
```
Comment (nastasi-oq) [inline installers/unix/env/install.sh:110]: `TOOLS` variable logic is not so intuitive, please add a comment explaining if 'n' or 'N'  => false else true

File: installers/unix/env/build.sh [hunk 2]
```diff
@@ -92,27 +99,38 @@ source pybuild/bin/activate
 rm -Rf oq-engine
 git clone -q --depth=1 -b $OQ_BRANCH https://github.com/gem/oq-engine.git
 
+rm -Rf oq-platform*
+git clone -q --depth=1 -b $OQ_BRANCH https://github.com/gem/oq-platform-standalone.git
+git clone -q --depth=1 -b $OQ_BRANCH https://github.com/gem/oq-platform-ipt.git
```
Comment (daniviga) [inline installers/unix/env/build.sh]: s/OQ_BRANCH/TOOLS_BRANCH/g

File: installers/unix/env/install.sh (hunk 1 above)
Comment (daniviga) [reply installers/unix/env/install.sh:110]: Done via 642abd5659f9b15ba1b9bd0ebda83f5f7d0ef7bb

**Comment by nastasi-oq** [comment]
LGTM

**Review by nastasi-oq — COMMENTED** [review]
LGTM

**Pull request merged**